site/Dance/intro.php

```diff
@@ -0,0 +1,8 @@
+
+
+    <div class="intro-text">
+      <p><?php print $introtext; ?></p>
+      <div class="intro-text-2">
+        <p><?php print $introtext; ?></p>
```

Comment:
I don't think you should be duplicating this text. What was the reason you have it here and also on line 4? Was this for responsive stuff?